--- a/etc/ironic/ironic.conf.sample
+++ b/etc/ironic/ironic.conf.sample
@@ -155,6 +155,7 @@
 # (string value)
 # Deprecated group/name - [DEFAULT]/logdir
 #log_dir = <None>
+log_dir=/var/log/ironic
 
 # Uses logging handler designed to watch file system. When log
 # file is moved or removed this handler will open a new log
@@ -730,6 +731,7 @@
 # Deprecated group/name - [DATABASE]/sql_connection
 # Deprecated group/name - [sql]/connection
 #connection = <None>
+connection=sqlite:////var/lib/ironic/ironic.db
 
 # The SQLAlchemy connection string to use to connect to the
 # slave database. (string value)
