--- a/etc/ironic/ironic.conf.sample
+++ b/etc/ironic/ironic.conf.sample
@@ -112,7 +112,7 @@
 # (Optional) The base directory used for relative --log-file
 # paths. (string value)
 # Deprecated group/name - [DEFAULT]/logdir
-#log_dir=<None>
+log_dir=/var/log/ironic
 
 # Use syslog for logging. Existing syslog format is DEPRECATED
 # and will be changed later to honor RFC5424. (boolean value)
@@ -609,7 +609,7 @@
 # Deprecated group/name - [DEFAULT]/sql_connection
 # Deprecated group/name - [DATABASE]/sql_connection
 # Deprecated group/name - [sql]/connection
-#connection=<None>
+connection=sqlite:////var/lib/ironic/ironic.db
 
 # The SQLAlchemy connection string to use to connect to the
 # slave database. (string value)
