--- a/etc/ironic/ironic.conf.sample
+++ b/etc/ironic/ironic.conf.sample
@@ -201,7 +201,7 @@
 # paths. This option is ignored if log_config_append is set.
 # (string value)
 # Deprecated group/name - [DEFAULT]/logdir
-#log_dir=<None>
+log_dir=/var/log/ironic
 
 # (Optional) Uses logging handler designed to watch file
 # system. When log file is moved or removed this handler will
@@ -809,7 +809,7 @@
 # Deprecated group/name - [DEFAULT]/sql_connection
 # Deprecated group/name - [DATABASE]/sql_connection
 # Deprecated group/name - [sql]/connection
-#connection=<None>
+connection=sqlite:////var/lib/ironic/ironic.db
 
 # The SQLAlchemy connection string to use to connect to the
 # slave database. (string value)
